test/integration/fn_addons.go

```diff
@@ -40,7 +38,8 @@ import (
 
 func testAddons(t *testing.T) {
 	t.Parallel()
-	client, err := pkgutil.GetClient()
+	p := "minikube"
```

Comment:
omit variable initialization for single-use strings - here, and elsewhere when possible.

But also - add a comment as to why this chooses a specific profile name.